lua/oil/config.lua

```diff
@@ -40,6 +40,10 @@ local default_config = {
   -- Set to true to autosave buffers that are updated with LSP willRenameFiles
   -- Set to "unmodified" to only save unmodified buffers
   lsp_rename_autosave = false,
+
+  -- The amount of time LSP waits for file operation before it errors
+  lsp_file_operation_timeout_ms = 1000,
+
```

Comment:
Since we're adding another option related to LSP file operations (and we may want more in the future), let's nest this under another table
```lua
lsp_file_methods = {
  timeout_ms = 1000
}
```

We'll also want to move `lsp_rename_autosave` into that table with a backwards compatibility shim and deprecation message, but I can do that after this PR.

Would also recommend rewording the comment to something like "Time to wait for LSP file operations to complete before skipping" to make it clear that we're waiting _on_ the LSP, and that the consequences of a timeout are only that it gets skipped.

Reply:
Hi @stevearc, I tried to do all of the above. Let me know if there is anything else I need to change. Thank you!